lib/main.js

```diff
@@ -40,11 +41,22 @@ function shim(Linter) {
   return LinterShim;
 }
 
+/**
+ * Checks whether or not the given grammar is for TypeScript
+ * @param {Grammar} grammar The grammar to check
+ * @return {boolean} Whether or not the grammar is for TypeScript
+ */
+function isTypescriptGrammar(grammar) {
+  return grammar.fileTypes.find(fileType => /.*ts/i.test(fileType));
```

Comment:
Could you elaborate what you mean by non-public API here? This is my first modification of an Atom package and am unaware of how else to do helper methods.